pkg/accounting/tally/tally.go

```diff
@@ -226,11 +193,11 @@ func (t *tally) Query(ctx context.Context) error {
 
 	//todo:  switch to bulk update SQL?
 	for k, v := range bwTotals {
-		nID := dbx.Granular_NodeId(k)
-		start := dbx.Granular_StartTime(lastBwTally.Value)
-		end := dbx.Granular_EndTime(latestBwa)
-		total := dbx.Granular_DataTotal(v)
-		_, err = tx.Create_Granular(ctx, nID, start, end, total)
+		nID := dbx.Raw_NodeId(k)
```

Comment:
We shouldn't mix database logic and service logic, it makes harder to follow the code. (Can be fixed in a separate PR).